.circleci/config.yml

```diff
@@ -306,60 +306,54 @@ jobs:
     docker:
       - image: circleci/php:7.4.15-apache-browsers
     steps:
+      - hosts_keyscan
       - checkout_code
-      - run:
-          name: Checkout edge branch
-          command: |
-            cd ~/project
-            git checkout -b $EDGE_BUILD_BRANCH
       - install_php_os_extensions
-      - install_platformsh_tools
+      - install_psh_cli
       - run:
-          name: Trigger a data sync from production environment to nightly edge build.
+          name: Trigger a data sync from production environment to an edge build.
           command: |
-            /home/circleci/.platformsh/bin/platform sync data -p $PLATFORM_PROJECT_ID -e $EDGE_BUILD_BRANCH -y
+            platform sync data -p $PLATFORM_PROJECT_ID -e $EDGE_BUILD_BRANCH -y
       - run:
           name: Turn off fastly module to allow for cleaner config import
           command: |
-            /home/circleci/.platformsh/bin/platform environment:drush pmu fastly -y -p $PLATFORM_PROJECT_ID -e $EDGE_BUILD_BRANCH
+            platform environment:drush pmu fastly -y -p $PLATFORM_PROJECT_ID -e $EDGE_BUILD_BRANCH
       - run:
           name: Refresh configuration as our db will contain active prod config after sync operation
           command: |
-            /home/circleci/.platformsh/bin/platform environment:drush cim -y -p $PLATFORM_PROJECT_ID -e $EDGE_BUILD_BRANCH
-      - run:
-          name: Run structure sync import all
-          command: |
-            /home/circleci/.platformsh/bin/platform environment:drush "ia --choice=safe" -y -p $PLATFORM_PROJECT_ID -e $EDGE_BUILD_BRANCH
+            platform environment:drush cim -y -p $PLATFORM_PROJECT_ID -e $EDGE_BUILD_BRANCH
       - run:
           name: Rebuild Taxonomy Entity Index
```

Comment:
Can't for the life of me remember what this is for, but it's working and it's been there for a while so.... if it works, don't fix it etc